datalad_next/create_sibling_webdav.py

```diff
@@ -422,41 +429,107 @@ def _create_sibling_webdav(
     # simplify downstream logic, export yes or no
     export_storage = 'export' in storage_sibling
 
+    existing_siblings = [r[1] for r in
+                         _yield_ds_w_matching_siblings(ds,
+                                                       (name, storage_name),
+                                                       recursive=False)
+                         ]
+
     if storage_sibling != 'no':
-        yield from _create_storage_sibling(
-            ds,
-            url,
-            storage_name,
-            existing,
-            credential,
-            export=export_storage,
-        )
+        if storage_name in existing_siblings:
+            yield from maybe_skip_sibling(credential, credential_name, ds,
+                                          existing, export_storage, name,
+                                          storage_name, storage_sibling,
+                                          url, 'storage')
+        else:
+            yield _create_storage_sibling(
+                    ds,
+                    url,
+                    storage_name,
+                    credential,
+                    export=export_storage,
+                    reconfigure=False,
+                )
+
     if 'only' not in storage_sibling:
-        yield from _create_git_sibling(
```

Comment:
The `yield from` pattern was desired, because the minimalistic implementation now has only one result per call, but this is hardly going to stay.

Reply:
Done